src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -110,6 +110,7 @@ for further details.
 	display: flex;
 	justify-content: space-between;
 	align-items: baseline;
+	margin-bottom: 32px;
```

Comment:
The `BreadcrumbBar` component is only used on this one layout so no chance of this shift cascading to other views.